tools/android_test.sh

```diff
@@ -42,5 +43,8 @@ fi
 # The steps are as follows.
 
 # $ meson build [flags...]
-# $ cd build
-# $ adb push res/ /data/local/tmp/nntr_android_test
+# meson build will unzip golden data for the unit tests
+cd ../../../
+meson build
```

Comment:
it would be nice to add some basic flags to get the proper golden data (e.g., `-Denable-fp16=true`, `-Denable-neon=true`, `-Denable-opencl=true`)

Reply:
Good point!! for more information we use below flags for build android_package

```
-Dplatform=android 
-Dopenblas-num-threads=1 
-Denable-tflite-interpreter=false 
-Denable-tflite-backbone=false 
-Denable-fp16=true 
-Denable-neon=true 
-Domp-num-threads=1 
-Denable-opencl=true 
-Dhgemm-experimental-kernel=false
```

Reply:
Thank you for the comments, all! It would be better to apply the same options for the android build. I will apply it 😀

Reply:
@DonghakPark I applied all the options you suggested except for the android option, for`meson.build` doesn't create resource for unittest with `-Dplatofrm=android` option

Please refer to 
https://github.com/nnstreamer/nntrainer/blob/63f987506ae8779bb31bf596c665b5fe99a95748/meson.build#L439

Reply:
Thank you for considering my opinion.
I think we need to ensure that testing is enabled in the future, even for cases of Android as well. (As there are increasingly more examples utilizing Android.)
I will create an issue for this matter. 

- #2775